src/walletConnect/saga.ts

```diff
@@ -183,9 +182,15 @@ function* createWalletConnectChannel() {
     }
   }) as EventChannel<WalletConnectActions>
 }
-
+/**
+ * It appears Actions.INITIALISE_PAIRING is emitting on both initial connections and on payload requests
```

Comment:
I'm not sure I understand this comment.  Can you rephrase it?  Specifically what/who is emitting?  Also, if what/who is emitting erroneously or unexpectedly, should we fix that instead?

Reply:
It looks like some old assumptions were made on our end [here](https://github.com/valora-inc/wallet/blob/main/src/walletConnect/walletConnect.ts#L50C3-L53) in `src/walleconnect/walletconnect.ts` about what what is a wallet connect action vs connection request. I agree a fix here is probably more appropriate.